<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GWidget</class>
 <widget class="QDialog" name="GWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Graph</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="generateButton">
           <property name="text">
            <string>Generate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="layoutButton">
           <property name="text">
            <string>Layouts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Nodes/Edges</string>
           </property>
           <property name="text">
            <string>0/0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearfindButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="GViewer" name="view"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="copysvgButton">
           <property name="text">
            <string>Copy SVG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savepngButton">
           <property name="text">
            <string>Save as PNG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fitscreenButton">
           <property name="text">
            <string>Fit screen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="actualsizeButton">
           <property name="text">
            <string>Actual Size</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GViewer</class>
   <extends>QGraphicsView</extends>
   <header location="global">cchelper.gviewer</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
